src/editor/components/components/AddLayerPanel/LayersOptions.js

```diff
@@ -1,50 +1,50 @@
 const LayersOptions = [
   {
-    value: 'Models: Personal Vehicles',
-    label: 'Models: Personal Vehicles',
-    mixinGroups: ['vehicles', 'vehicles-rigged'],
-    onClick: () => console.log('Models: Personal Vehicles')
+    value: 'Street Layers',
+    label: 'ꈨ Street Layers',
```

Comment:
I just see an empty square or a square with ? on Ubuntu 22.04 Chrome for this one.